content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,253 @@
+    "max_features": 200,
+    "ngrams": 2
+  },
+  "train": {
+    "seed": 20170428,
+    "n_est": 50,
+    "min_split": 0.01
+  }
+}
+```
+
+## Example: Filtering with `stages`.
+
+> Working on https://github.com/iterative/example-get-started
+
+`stages` can a single string:
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(stages="prepare")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "prepare": {
+    "split": 0.2,
+    "seed": 20170428
+  }
+}
+```
+
+Or an iterable of strings:
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(stages=["prepare", "train"])
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "prepare": {
+    "split": 0.2,
+    "seed": 20170428
+  },
+  "train": {
+    "seed": 20170428,
+    "n_est": 50,
+    "min_split": 0.01
+  }
+}
+```
+
+## Example: Using `rev`.
+
+> Working on https://github.com/iterative/example-get-started
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(rev="tune-hyperparams")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "prepare": {
+    "split": 0.2,
+    "seed": 20170428
+  },
+  "featurize": {
+    "max_features": 200,
+    "ngrams": 2
+  },
+  "train": {
+    "seed": 20170428,
+    "n_est": 100,
+    "min_split": 8
+  }
+}
+```
+
+## Example: Using `targets`.
+
+> Working on `multi-params-files` folder of
+> https://github.com/iterative/pipeline-conifguration
+
+You can pass a single target:
+
+```py
+import json
+import dvc.api
+params = dvc.api.params_show("params.yaml")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "run_mode": "prod",
+  "configs": {
+    "dev": "configs/params_dev.yaml",
+    "test": "configs/params_test.yaml",
+    "prod": "configs/params_prod.yaml"
+  },
+  "evaluate": {
+    "dataset": "micro",
+    "size": 5000,
+    "metrics": ["f1", "roc-auc"],
+    "metrics_file": "reports/metrics.json",
+    "plots_cm": "reports/plot_confusion_matrix.png"
+  }
+}
+```
+
+Or multiple targets:
+
+```py
+import json
+import dvc.api
+params = dvc.api.params_show(
+  "configs/params_dev.yaml", "configs/params_prod.yaml")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "configs/params_prod.yaml:run_mode": "prod",
+  "configs/params_prod.yaml:config_file": "configs/params_prod.yaml",
+  "configs/params_prod.yaml:data_load": {
+    "dataset": "large",
+    "sampling": {
+      "enable": true,
+      "size": 50000
+    }
+  },
+  "configs/params_prod.yaml:train": {
+    "epochs": 1000
+  },
+  "configs/params_dev.yaml:run_mode": "dev",
+  "configs/params_dev.yaml:config_file": "configs/params_dev.yaml",
+  "configs/params_dev.yaml:data_load": {
+    "dataset": "development",
+    "sampling": {
+      "enable": true,
+      "size": 1000
+    }
+  },
+  "configs/params_dev.yaml:train": {
+    "epochs": 10
+  }
+}
+```
+
+## Example: Git URL as `repo`.
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(
+    repo="https://github.com/iterative/demo-fashion-mnist")
```

Comment:
This one also needs some explanation/motivation I think. Or if it's basically the same as the others, we probably don't need the example 🙂 (the motivation can be in the **Parameters** section).

Reply:
Motivation is that you can use the API to retrieve params from a repository without having to clone and/or navigate to it locally